compiler-rt/lib/radsan/tests/radsan_test_utilities.h

```diff
@@ -14,7 +14,8 @@
 namespace radsan_testing {
 
 template <typename Function>
-[[clang::realtime]] void realtimeInvoke(Function &&func) {
+void realtimeInvoke(Function &&func) [[clang::nonblocking]]
```

Comment:
Notice we must now have this after the declaration, before the declaration throws a warning and does not work

Reply:
Nice - yep, this is good